insights/specs/default.py

```diff
@@ -330,6 +330,7 @@ class DefaultSpecs(Specs):
     libssh_server_config = simple_file("/etc/libssh/libssh_server.config")
     libvirtd_log = simple_file("/var/log/libvirt/libvirtd.log")
     limits_conf = glob_file(["/etc/security/limits.conf", "/etc/security/limits.d/*.conf"])
+    localectl = simple_command("/usr/bin/localectl")
```

Comment:
Better change the spec name and command more specifically as 
```
localectl_status = simple_command("/usr/bin/localectl --status")
```

Reply:
/usr/bin/localectl: unrecognized option '--status'

Update as localectl_status = simple_command("/usr/bin/localectl status")